frontend/views/utils/markdown-utils.js

```diff
@@ -40,14 +40,15 @@ export function renderMarkdown (str: string): any {
       entryText = entryText.replace(/</g, '&lt;').replace(/>/g, '&gt;')
 
       entryText = entryText.replace(/\n(?=\n)/g, '\n<br>')
-        .replace(/<br>\n(\s*)(\d+\.|-)/g, '\n\n$1$2') // custom-handling the case where <br> is directly followed by the start of ordered/unordered lists
-        .replace(/(\d+\.|-)(\s.+)\n<br>/g, '$1$2\n\n') // this is a custom-logic added so that the end of ordered/un-ordered lists are correctly detected by markedjs.
+        .replace(/<br>\n(\s*)(\d+\.|-)/g, '\n\n$1$2') // [1] custom-handling the case where <br> is directly followed by the start of ordered/unordered lists
+        .replace(/(\d+\.|-)(\s.+)\n<br>/g, '$1$2\n\n') // [2] this is a custom-logic added so that the end of ordered/un-ordered lists are correctly detected by markedjs.
```

Comment:
As inline-commented in the same file, `markedjs` with `gfm` style has a default behaviour where it collapses multiple consecutive line-breaks into one which is not what we want for the app. `[1]` and `[2]` here are the custom logics added to work around this. But [2] here didn't capture an edge-case where the **last-child of a list** ends with `<code>` element.

(The reason for this edge case is that, we split the whole text by markdown code elements first before applying these `[1]` and `[2]` because no content wrapped within `<code>...</code>` should be converted.)

So added a fix in `L51` below to take this into account.